kernel_gateway/services/cell/parser.py

```diff
@@ -1,26 +1,64 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
+"""Parser for notebook cell API annotations."""
+
 import re
 import sys
 
 def first_path_param_index(endpoint):
-    '''Returns the index to the first path parameter for the endpoint. The
+    """Gets the index to the first path parameter for the endpoint. The
     index is not the string index, but rather where it is within the path.
```

Comment:
The returned value is not the string index, but rather the depth of where the endpoint is found in the the path.